### PR TITLE
Support placeholder in service fabric services names

### PR DESCRIPTION
**Pull request by FelixBoers**
Fixes / New Feature #
#721

### CODE REVIEW

**Comment by FelixBoers** [comment]
Ready for review! @TomPallister

File: test/Ocelot.UnitTests/DownstreamUrlCreator/DownstreamUrlCreatorMiddlewareTests.cs [hunk 1]
```diff
@@ -223,7 +223,7 @@ public void should_create_service_fabric_url()
             this.Given(x => x.GivenTheDownStreamRouteIs(downstreamRoute))
                 .And(x => GivenTheServiceProviderConfigIs(config))
                 .And(x => x.GivenTheDownstreamRequestUriIs("http://localhost:19081"))
-                .And(x => x.GivenTheUrlReplacerWillReturn("/api/products/1"))
+                .And(x => x.GivenTheUrlReplacerWillReturn("Ocelot/OcelotApp/api/products/1"))
```
Comment (TomPallister) [inline test/Ocelot.UnitTests/DownstreamUrlCreator/DownstreamUrlCreatorMiddlewareTests.cs]: Is there a reason to add ```Ocelot/OcelotApp``` to all these tests

Comment (FelixBoers) [reply test/Ocelot.UnitTests/DownstreamUrlCreator/DownstreamUrlCreatorMiddlewareTests.cs]: Thanks for your response.
The reason is, that `_replacer.Replace()` now gets called twice. Once for the `context.DownstreamReRoute.DownstreamPathTemplate` and once for the ServiceFabric service name.

I've just updated the commit to make it more clear. Now it looks like follows:

```csharp
.And(x => x.GivenTheUrlReplacerWillReturnSequence("/api/products/1", "Ocelot/OcelotApp"))
```
The first parameter `/api/products/1` gets returned when replacing the placeholders in `context.DownstreamReRoute.DownstreamPathTemplate`. 
The second parameter `Ocelot/OcelotApp` when replacing the placeholders in the ServiceFabric service name

**Comment by FelixBoers** [comment]
@TomPallister Who is able to review this pull request? Can I help somehow?

**Comment by FelixBoers** [comment]
@philproctor Thanks in advance. Let me know if there is anything I can do.

**Comment by philproctor** [comment]
@TomPallister @briansantura @thiagoloureiro Anyone want to add a second review? Looks like tests are there and I don't see any issues, outside of a minor performance impact for the replacement on service fabric requests

**Comment by thiagoloureiro** [comment]
@philproctor I think we are good to merge

**Pull request merged**